src/frontend/src/icp/components/fee/EthereumEstimatedFee.svelte

```diff
@@ -10,22 +10,17 @@
 		ETHEREUM_FEE_CONTEXT_KEY,
 		type EthereumFeeContext
 	} from '$icp/stores/ethereum-fee.store';
-	import { isTokenCkErc20Ledger } from '$icp/utils/ic-send.utils';
 	import { ethereumFeeTokenCkEth } from '$icp/derived/ethereum-fee.derived';
-	import { tokenWithFallbackAsIcToken } from '$icp/derived/ic-token.derived';
 	import type { Token } from '$lib/types/token';
 	import FeeAmountDisplay from '$icp-eth/components/fee/FeeAmountDisplay.svelte';
 
-	let ckEr20 = false;
-	$: ckEr20 = isTokenCkErc20Ledger($tokenWithFallbackAsIcToken);
-
 	const { store } = getContext<EthereumFeeContext>(ETHEREUM_FEE_CONTEXT_KEY);
 
 	let feeToken: Token;
-	$: feeToken = ckEr20 ? $ethereumFeeTokenCkEth ?? $ckEthereumNativeToken : $ckEthereumNativeToken;
+	$: feeToken = $ethereumFeeTokenCkEth ?? $ckEthereumNativeToken;
 
 	let maxTransactionFee: bigint | undefined | null = undefined;
-	$: maxTransactionFee = $store?.maxTransactionFee;
+	$: maxTransactionFee = $store?.maxTransactionFee ?? 100_000_000_000_000_000n;
```

Comment:
What's `100_000_000_000_000_000n`?

Reply:
oh wow, sorry, it was just a test I was doing, shouldn't be there

Reply:
I updated with a Staging screenshot